spec/workers/sidekiq_cron_jobs_spec.rb

```diff
@@ -33,7 +33,7 @@
   context 'when WikiEd Feature disabled' do
     before { allow(Features).to receive(:wiki_ed?).and_return(false) }
 
-    it 'run default campaign update' do
+    it 'do not run default campaign update' do
```

Comment:
I just fixed this, since the description was incorrectly saying "run default campaign update", when what I wanted to specified was the opposite.